packages/landing/components/index.ts

```diff
@@ -0,0 +1,3 @@
+import { Navbar } from "./navbar/Navbar";
+
+export { Navbar };
```

Comment:
We don't need this. My suggestion works in the same way.

```suggestion
export * from "./navbar";
```

Reply:
Ok!